doc/classes/SpriteFrames.xml

```diff
@@ -75,7 +75,7 @@
 				[codeblock]
 				absolute_duration = relative_duration / (animation_fps * abs(speed_scale))
 				[/codeblock]
-				In this example, [code]speed_scale[/code] refers to either [member AnimatedSprite2D.speed_scale] or [member AnimatedSprite3D.speed_scale].
+				In this example, [code]speed_scale[/code] refers to either [member AnimatedSprite2D.playback_speed] or [member AnimatedSprite3D.playback_speed].
```

Comment:
```suggestion
				In this example, [code]playback_speed[/code] refers to either [member AnimatedSprite2D.playback_speed] or [member AnimatedSprite3D.playback_speed].
```

Here and in line 76 should be renamed too.

Reply:
It's just an alias, so I don't think there's anything wrong with it.